app/models/email.rb

```diff
@@ -41,6 +41,13 @@ def persisted?
 
   private
 
+  def get_agreement_letter_attachment
+    {
+      name: (I18n.t 'emails.agreement_letter_attachment'),
+      content: File.read(Rails.root.join('storage', 'agreement_letters', 'real_agreement_letter.pdf'))
```

Comment:
I'd suggest to read the path from the config, either in this PR, or later (we need a technical debt issue then)